src/frontend/shared-components/src/components/table/Table.js

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import "./Table.css";
+
+const TableElement = ({ element: { name, value } }) => {
+  return (
+    <tr>
+      <td>{name}</td>
+      <td>{value}</td>
+    </tr>
+  );
+};
+
+export default function Table({
+  table: { id, header, tableElements, tableStyle }
```

Comment:
can this just be:
```suggestion
  table: { id, header, elements, style }
```
since it is already within `table`?

Reply:
nit: also, can we change `header` to something like `heading` or `label` so it is in no way confused with the actual header component? I know they're unrelated components but would be nice